README.md

```diff
@@ -87,34 +91,37 @@ cache @matchdefault {
-| `stale`                 | The stale duration                                                                                                                           | `25m`                                                                                                                   |
-| `ttl`                   | The TTL duration                                                                                                                             | `120s`                                                                                                                  |
-| `log_level`             | The log level                                                                                                                                | `One of DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL it's case insensitive`                                           |
+|  Key                      |  Description                                                                                                                                 |  Value example                                                                                                          |
+|:--------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
+| `api`                     | The cache-handler API cache management                                                                                                       |                                                                                                                         |
+| `api.basepath`            | BasePath for all APIs to avoid conflicts                                                                                                     | `/your-non-conflict-route`<br/><br/>`(default: /souin-api)`                                                             |
+| `api.prometheus`          | Enable the Prometheus metrics                                                                                                                |                                                                                                                         |
+| `api.souin.basepath`      | Souin API basepath                                                                                                                           | `/another-souin-api-route`<br/><br/>`(default: /souin)`                                                                 |
+| `api.souin.security`      | Enable JWT validation to access the souin endpoint                                                                                           | `(default: false)`                                                                                                      |
```

Comment:
This should be adjusted as well.

I think this option should probably be called `validate_jwt` rather than `security`, it's a bit meaningless as an option name.

Why isn't this on by default, btw? You would think that any security options should be enabled by default.

Reply:
In the past I planned to write a basic auth too. But now that Souin is designed to work behind a RP like caddy/traefik or an API Gateway, I don't think this feature is really useful. I think that users would setup the auth directly in their proxy instead of only on Souin. I'm a bit lost about this feature.
I'll support both `security` and `validate_jwt` to avoid BC in the core and deprecate the `security` directive.

Reply:
Yeah; as much as possible I think the cache module should only do caching, and as much as possible should let authentication be configured elsewhere.